src/commands/time/finish.js

```diff
@@ -26,102 +26,106 @@ function controller(t) {
         var isTask = !isBug && e.tp_task;
 
         utils.log('    Logging time on target process...');
-        if(isBug) {
-          utils.log('    Bugs are configured to be logged against: ' + tp.bugTimeBehavior);
-        }
+        tp.getBug(e.tp_task.id).then(function (result) {
+          tp.bugTimeBehavior = tp.getCustomField(result.Project.Id).then(function(value) {
+            if(isBug) {
+                utils.log('    Bugs are configured to be logged against: ' + value.items[0].issueCFRaw);
```

Comment:
It would make more sense for `getCustomField` to return what is in `items[0].issueCFRaw`, rather than having to access like this each time we want to use the value.